pset-functions.js

```diff
@@ -135,6 +215,53 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+let word = "cabbage"
+let totalScore = 0
+let score = 0
+//let onePoint = ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"  ]
+
+function scrabbleScore(){
+   return word.split("")
+}
+
+    console.log(scrabbleScore())
+
+for (let i = 0; i < word.length; i++) {
+
+    //A, E, I, O, U, L, N, R, S, T       1
+    if(scrabbleScore()[i] === "a" || scrabbleScore()[i] === "e" || scrabbleScore()[i] === "i" || scrabbleScore()[i] === "o" || scrabbleScore()[i] === "u" ||
+    scrabbleScore()[i] === "l" || scrabbleScore()[i] === "n" || scrabbleScore()[i] === "r" || scrabbleScore()[i] === "s" || scrabbleScore()[i] === "t") {
+      score++
+    }
+    //D, G                               2
+    else if (scrabbleScore()[i] === "d" || scrabbleScore()[i] === "g") {
+      score += 2
+    }
+    //B, C, M, P                         3
+    else if (scrabbleScore()[i] === "b" || scrabbleScore()[i] === "c" || scrabbleScore()[i] === "m" || scrabbleScore()[i] === "p") {
+      score += 3
+    }
+    //F, H, V, W, Y                      4
+    else if (scrabbleScore()[i] === "f" || scrabbleScore()[i] === "h" || scrabbleScore()[i] === "v" || scrabbleScore()[i] === "n" || scrabbleScore()[i] === "y") {
+      score += 4
+    }
+    //K                                  5
+    else if (scrabbleScore()[i] === "k"){
+      score += 5
+    }
+    //J, X                               8
+    else if (scrabbleScore()[i] === "j" || scrabbleScore()[i] === "x") {
+      score += 8
+    }
+    //Q, Z                               10
+    else if (scrabbleScore()[i] === "Q" || scrabbleScore()[i] === "Z") {
+      score += 10
+    }
+
+  }
+
+console.log(score)
+
```

Comment:
@jl1982 Good Job on this problem here. Looping through each letter as you split the word into an array. Good job with your conditionals to check for each letter and increment the value of `score` by the point for the letter. One suggestion for line 257 you should keep the string ex. q and z should be lowercase. Since all of the other letter is lowercase you would want to keep them all consistent.